egs2/sdsv21/spk1/local/generate_trial_file.py

```diff
@@ -0,0 +1,44 @@
+# generate_trial_file.py
```

Comment:
can you add a random seed to make it reproducible?

Reply:
Could you kindly show me what part of this would be random/reproducible? The trials are from all the combinations, not a sub sample.